architecture/core_concepts/routes.adoc

```diff
@@ -605,3 +605,12 @@ termination.
 validate the endpoint certificate, securing the connection from the router to
 the destination. This field is required, but only for re-encryption.
 ====
+
+
+[[router-sharding]]
+== Router Sharding
+
+In OpenShift, router sharding is implemented via
+link:../../architecture/core_concepts/pods_and_services.html#labels[labels]
+in the router's `metadata` field.
+These labels can be used to specialize the service selector.
```

Comment:
I feel like this could really benefit from an example. I'm actually not following this at all. Also, does "service selector" = "node selector"?